packages/inspection-capture-web/src/PhotoCapture/PhotoCaptureHUD/PhotoCaptureHUDElementsSight/SightGuideline/SightGuideline.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useMemo, useState } from 'react';
+import { Button } from '@monkvision/common-ui-web';
+import { CaptureAppConfig } from '@monkvision/types';
+import { useTranslation } from 'react-i18next';
+import { getLanguage } from '@monkvision/common';
+import { usePhotoCaptureHUDButtonBackground } from '../../hooks';
+import { styles } from './SightGuideline.styles';
+
+/**
+ * Props of the SightGuidelineButton component.
+ */
+export interface SightGuidelineProps
+  extends Pick<CaptureAppConfig, 'enableAddDamage' | 'sightGuidelines' | 'enableSightGuidelines'> {
+  /**
+   * The id of the sight.
+   */
+  sightId: string;
+}
+
+/**
+ * Custom button displaying the sight guideline.
+ */
+export function SightGuideline({
+  sightId,
+  sightGuidelines,
+  enableSightGuidelines,
+  enableAddDamage,
+}: SightGuidelineProps) {
+  const [showGuideline, setShowGuideline] = useState(true);
+  const primaryColor = usePhotoCaptureHUDButtonBackground();
+  const { i18n } = useTranslation();
+
+  const style = useMemo(
+    () => (enableAddDamage ? styles['container'] : styles['containerWide']),
+    [enableAddDamage, styles],
+  );
+
+  const guideline = useMemo(() => {
+    const guidelineFound = sightGuidelines?.find((value) => value.sightIds.includes(sightId));
+    return guidelineFound ? guidelineFound[getLanguage(i18n.language)] : undefined;
+  }, [sightId, sightGuidelines]);
```

Comment:
I don't think you need to memoize these values, the computation is not that expensive